src/globals/scss/core/_typography.scss

```diff
@@ -6,12 +6,8 @@
 
     display: block;
 
-    margin-top: 0;
-    margin-bottom: $govuk-spacing-scale-5;
-
-    @include mq($from: tablet) {
-      margin-bottom: $govuk-spacing-scale-7;
-    }
+    margin-top: govuk-spacing-unit($govuk-spacing-scale-0);
```

Comment:
I think the benefit from having 0 in the 'scale' is that we make sure we generate overrides for it, but it seems kinda weird to add this extra complexity when `margin-top: 0` should (?) always have the same effect? Thoughts?

Reply:
Agreed. I discussed with @dashouse and he saw no real benefit here either. Have removed all zero variables for margin/padding from style declarations.